api-spec/filters.md

```diff
@@ -0,0 +1,133 @@
+STAC Core is intending to provide a minimal set of features so that there is a low bar to entry for catalog
+implementations. The core concepts in STAC are Spatial and Temporal data, so those are the only filters we want
+to support in Core.
+
+Filters should be as simple as possible for a client to construct. Since we are returning json, we want to also
+consume json. This allows clients to support filtering without additional tools. So the core filters will be 
+JSON based. The search enpoint will accept application/json format queries, and GET on the collection will 
+support URL encoded JSON. POST search the is recommended way to filter results to avoid the URL encoding 
+issues that can happen with GET.
```

Comment:
Maybe say that POST is required, and we recommend GET as well. 

We also might call out how a POST makes sense in REST - that it's against a 'search' endpoint, not the items themselves. Perhaps not at the top, but somewhere (could also make the recommendation on POST lower).

Reply:
I took a stab at adding that to the document. I'm thinking I'll make a few small changes here, then merge so others can contribute going forward. It's hard for others to make changes to my feature branch!